modules/test/playwright/playwright.config.ts

```diff
@@ -5,14 +5,19 @@
 
 import {defineConfig} from '@playwright/test';
 
-import {config as setup} from './tests/global.setup.config';
-import {config as object} from './tests/object-web/config';
-import {config as portalWeb} from './tests/portal-web/config';
-import {config as usersAdminWeb} from './tests/users-admin-web/config';
+import {config as setupConfig} from './tests/global.setup.config';
```

Comment:
Maybe we should convert setup to another project (since it is in fact a project) with its own folder. That way: (1) we can store more files associated to setup and (2) we can name it as the rest of the configs.

Reply:
We can leave it for another PR, in any case...

Reply:
Makes sense to me. "setup" is also pretty vague, unless we want to pile in more to login. Something like "login" maybe makes more sense. Also, some tests might require user to be logged out.

In any case, I'll remove the change on this file. It is the main point of conflicts for in-flight PR. We can rename this when it's more stable.